inst/sitevisitpeak/sitevisitpeak.Rmd

```diff
@@ -1,14 +1,16 @@
 ---
-output: portrait_document
-classoption: legalpaper, landscape, fontsize=10pt
+output: html_document
+classoption: legalpaper, landscape, fontsize=8pt
 geometry: top=.5cm, bottom=.5cm, left=1cm, right=1cm
 ---
 
 ![](http://www.usgs.gov/newsroom/images/2015-04-07/usgs_logo.png)
 
-## `r getReportMetadata(ts,'title',required=TRUE)` Report
+**`r getReportMetadata(ts,'title',required=TRUE)` Report**
 
-**Location:**  `r getReportMetadata(data,'stationId',required=TRUE)` `r getReportMetadata(data,'stationName',required=TRUE)`
+**Location:**  `r getReportMetadata(data,'locationNumber',required=TRUE)` `r getReportMetadata(data,'locationName',required=TRUE)`
+
+**Sub-Location:** `r getReportMetadata(data,'sublocation',required=TRUE)`
```

Comment:
from the mockup and the jira ticket I see this in the header @thongsav-usgs 

https://internal.cida.usgs.gov/jira/browse/AQCU-265